make/config/projectcontour/contour-gateway.yaml

```diff
@@ -5049,8 +5039,8 @@ spec:
   strategy:
     type: RollingUpdate
     rollingUpdate:
-      # This value of maxSurge means that during a rolling update
-      # the new ReplicaSet will be created first.
+      #! This value of maxSurge means that during a rolling update
+      #! the new ReplicaSet will be created first.
```

Comment:
`#` in `ytt` is the default marker for a `ytt` expression (like [here](https://github.com/cert-manager/cert-manager/pull/5104/commits/6cb826833eb79ab0ce930ea8a4138b2a0994df3a#diff-255f2a83fb65076ca89470bc45e65be11f2efd21cb5e4665067c18a80eca9e19R5026) so by default it complains about `#` without `@` (reasoning [here](https://carvel.dev/ytt/docs/v0.39.0/file-marks/#type-detection-for-yaml-files)) and `#!` can instead be used for comments
We could have used `--ignore-unknown-comments` flag to not have it complain, but since there aren't a lot of comments in this file I preferred to use `#@` for it to check whether we haven't forgotten the `@` in `#@`